content/events-api/subscription.md

```diff
@@ -78,23 +78,34 @@ This request is JSON-based (`Content-Type: application/json`) and contains wrapp
 | `data` | Contains the resource data. Learn more about the data wrapper, including its JSON schema, on our [Events API Reference](/events-reference/events-reference-serenity/products.html). | | |
 
 
-## Debugging events
+## Debug events
+
+::: warning
+The following features are only available for **Serenity** customers.
```

Comment:
That's not true, isn't it ?
Technically, it's a feature only present on Serenity, because it's only on master for now.
It's not an exclusivity to Serenity customers.

Reply:
=> available for our CE and EE SaaS customers? 
Indeed, it will be available in the 6.0

Reply:
available for our CE and EE SaaS customers or 6.0 ?

Reply:
the 6.0 is not released yet ;)